.gitmodules

```diff
@@ -77,3 +77,6 @@
 [submodule "third_party/onnx"]
 	path = third_party/onnx
 	url = https://github.com/onnx/onnx.git
+[submodule "aten/src/ATen/cpu/sleef"]
```

Comment:
Can we start adding these libraries to top-level `third_party` instead of inside of ATen?